app/code/Magento/MediaGalleryApi/Model/Asset/Command/DeleteByPathInterface.php

```diff
@@ -10,14 +10,14 @@
 
 /**
  * A command represents the media gallery asset delete action. A media gallery asset is filtered by path value.
+ * @api
  */
 interface DeleteByPathInterface
```

Comment:
It's not clear how DeleteByPathInterface differs from DeleteByDirectoryPathInterface
and how DeleteByPathInterface will work if Directory path would be provided as an argument. 

We either need  1 generic interface or 2 interfaces with more explicit names to understand the difference

Reply:
DeleteByPathInterface will not delete the asset unless there is a strict match of the path. It makes it much safer to use this service for single asset deletion
DeleteByDirectoryPathInterface does not require strict path match, wildcard match is used allowing to delete the set of assets by directory path in a single request

Reply:
Interfaces were merged